extensions/amp-3q-player/0.1/test/test-amp-3q-player.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
@JuliusThms Most likely caused by component not firing `VideoEvents.LOAD` when iframe is ready. In `layoutCallback` do the following instead:

```
return this.playerReadyResolver_.then(() => {
   this.element.dispatchCustomEvent(VideoEvents.LOAD);
});
```